services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/model/MongoDbBasedCredentialsDao.java

```diff
@@ -50,15 +50,19 @@
  */
 public final class MongoDbBasedCredentialsDao extends MongoDbBasedDao implements CredentialsDao, HealthCheckProvider {
 
-    private static final Logger LOG = LoggerFactory.getLogger(MongoDbBasedCredentialsDao.class);
-
-    private static final String KEY_AUTH_ID = String.format("%s.%s", CredentialsDto.FIELD_CREDENTIALS,
+    public static final String KEY_AUTH_ID = String.format("%s.%s", CredentialsDto.FIELD_CREDENTIALS,
```

Comment:
Is there a reason why you made all these field public? Usage in my IDE shows only this class...

Reply:
Some of them are used by the PROJECTION constant which is declared public. So the used constants need to be declared before that one. And since the PROJECTION is public and we want to stick with the declaration order (public, protected, private), they also need to be public ...